docs/wireframe/admin/07-batch-list.html

```
@@ -0,0 +1,279 @@
+        </div>
+      </div>
+
+      <!-- Today's Issues -->
+      <div class="today-issues">
+        <div class="today-issues-title">
+          <span>⚠️</span> 오늘의 이슈 (3건)
+        </div>
+        <div class="issue-item">
+          <span class="issue-severity warning">WARNING</span>
+          <div class="issue-content">
+            <div class="issue-job">krDailyPriceJob > priceCollectStep</div>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)